.gitignore

```diff
@@ -28,7 +28,11 @@ website/node_modules
 *.iml
 
 website/vendor
+vendor
+.vscode/
 
+# makefile
```

Comment:
Do we need to ignore those (lines: 34-35)?

Reply:
Not sure, probably best to remove those lines from the .gitignore